api/params.go

```diff
@@ -701,6 +701,7 @@ func (p *Params) SetDefaults(gitSource, gitOwner, gitName, appLabel, buildVersio
 			"190.93.240.0/20",
 			"197.234.240.0/22",
 			"198.41.128.0/17",
+			"104.24.0.0/14",
```

Comment:
@afarid if it's changeable I would suggest to have method the list the cloud ip form this line https://www.cloudflare.com/ips-v4 and return []string so it will be dynamic and no need to bother again :)

Reply:
Yes, This is better,  This what discussed with @snigdhasambitak, I will create another PR for this.

Reply:
@msharbaji  could you please merge this pr :) ?